packages/deploy/release-script/release-script.ts

```diff
@@ -164,21 +157,63 @@ const getReleaseConfig = async (releaseType: ReleaseType): Promise<ReleaseConfig
       await executeAndLog(`git push upstream ${gitTag}`)
     }
   }
+  // **** Cleanup ****
 
-  // **** Dealing with dist ****
+  await executeAndLog('git reset --hard HEAD')
 
-  if (dryRun) {
-    console.log('Skipping build because dry-run')
-  } else {
-    // For test it's pain to wait for build as you can test build separately.
-    await executeAndLog('npm run build')
-  }
+  console.log(chalk.green('Released - 😎 GLORIOUS SUCCESS 😎'))
+}
 
-  await executeAndLog(`npm publish --tag=${distTag}${dryRun ? ' --dry-run' : ''}`)
+const simplePrompt = async <T> (question: DistinctQuestion<T>): Promise<T> => {
+  const result = await inquirer.prompt({
+    name: 'question',
+    ...question,
+  })
+  // Inquirer typing is pure pain.
+  return (result as unknown as { question: T }).question
+}
+const inquireReleaseType = async () => simplePrompt<ReleaseType>({
+  type: 'list',
+  message: 'Select build type',
+  choices: ['large', 'tiny', 'next', 'experimental'],
+  default: 'experimental',
+})
+const inquireDryRun = async () => simplePrompt<boolean>({
+  type: 'confirm',
+  default: false,
+  message: 'Do you want a dry-run? (there won\'t be any non reversible changes)',
+})
+const checkIfTooLate = async (releaseType: ReleaseType) => {
+  const result = await simplePrompt<boolean | undefined>({
+    type: 'confirm',
+    default: false,
+    when: () => new Date().getHours() > 20 && ['large', 'tiny'].includes(releaseType),
+    message: 'It\'s too late already, sure you\'re up for release at this time?',
+  })
+  if (result === undefined) {
+    return true
+  }
+  return result
```

Comment:
```suggestion
 return result === undefined ? true : result
```

Reply:
I generally prefer early return to ternary. )